docs/layouts/shortcodes/generated/security_configuration.html

```diff
@@ -134,6 +134,12 @@
             <td>String</td>
             <td>The secret to decrypt the keystore file for Flink's for Flink's internal endpoints (rpc, data transport, blob server).</td>
         </tr>
+        <tr>
+            <td><h5>security.ssl.internal.keystore-type</h5></td>
+            <td style="word-wrap: break-word;">"pkcs12"</td>
```

Comment:
You're right. Since I generated the documentation with Java 11, it output PKCS12. Do you know how I can stop the default value in the documentation from being generated and instead add it manually?

Reply:
I found an `OverrideDefault` annotation, will try that

Reply:
updated documentation